setup.py

```diff
@@ -68,3 +68,16 @@
               'bubblegum.xrf', 'bubblegum.xrf.model',
               'bubblegum.xrf.view'],
 )
+
+import shutil
+import os
+
+try:
+    os.mkdir(os.path.join(os.path.expanduser('~'), '.bubblegum'))
```

Comment:
This sort of stuff really steps on the conda install procedure.  If we are going to write into users home directories we should follow XDG (http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html).

This should be
```python
config_dir = os.path.join(os.path.expanduser('~'), '.config/bubblegum')
```

I think that for now it is better to bundle the 'default' configs with the source and deal with reading from user-supplied files later.